pkg/daemon/tasks.go

```diff
@@ -58,38 +62,90 @@ func (d *Daemon) listTasksHandler(engine api.Engine) func(w http.ResponseWriter,
 
 		cr, _ := engine.RunnerByName("cluster:k8s")
 		rr := cr.(*runner.ClusterK8sRunner)
-		allocatableCPUs, allocatableMemory, _ := rr.GetClusterCapacity()
 
-		_, _ = w.Write([]byte("<strong>cluster resources</strong><br/>"))
-		_, _ = w.Write([]byte(fmt.Sprintf("capacity cpus: %d<br/>", allocatableCPUs)))
-		_, _ = w.Write([]byte(fmt.Sprintf("capacity memory: %s<br/>", humanize.Bytes(uint64(allocatableMemory)))))
+		var allocatableCPUs, allocatableMemory int64
+		if rr.Enabled() {
```

Comment:
This is something worth refactoring in the future. Not all Testground daemons will support all runners, and right now we don't have a good way of defining which runners a given deployment supports, i.e. which runners are enabled.

---

The purpose of this bit here is to make sure that we show relevant information when Testground is working in cluster mode (i.e. on TaaS), and hide this information when running Testground locally and we don't have a cluster available.